core/modules/node/node.entity.inc

```diff
@@ -462,7 +478,14 @@ class NodeStorageController extends EntityDatabaseStorageController {
       }
       else {
         $op = 'update';
-        $return = backdrop_write_record($this->entityInfo['base table'], $entity, $this->idKey);
+        // Update the base node table, but only if this revision is marked as
+        // the default.
+        if ($entity->isDefaultRevision()) {
+          $return = backdrop_write_record($this->entityInfo['base table'], $entity, $this->idKey);
+        }
+        else {
+          $return = SAVED_REVISION;
```

Comment:
This may cause API breaks in some situations. Instead of a new return value, I think it may be preferable to return `SAVED_UPDATED` as before, even though we're not updating the entity base table.

This also would make it so that nodes wouldn't have a special 4th return value when compared to other types of entities.